cleverhans/picklable_model.py

```diff
@@ -575,3 +575,114 @@ def fprop(self, x, dropout=False, dropout_dict=None, **kwargs):
         if dropout:
             return tf.nn.dropout(x, include_prob)
         return x
+
+
+class Residual(Layer):
+    """A residual network layer that uses group normalization."""
+
+    def __init__(self, out_filter, stride, activate_before_residual=False):
+        self.__dict__.update(locals())
+        del self.self
+        self.lrelu = LeakyReLU(0.1)
```

Comment:
Would it make sense to let users control the leaky relu parameter through an arg to the constructor of `Residual`?